src/devices/Sn74hc595/Sn74hc595.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Do you actually require netcoreapp3.1? In the end this will ship in Iot.Device.bindings which currently targets netcoreapp2.1, so  I'm assuming you don't depend on 3.1 or build would be failing right now so could you move back to 2.1 here?

Reply:
Good call. will fix.

Reply:
I left the sample app as 3.1. I think that will help usability. Thoughts?

Reply:
I'm fine with that, we only really cared about the other one because the project in that case isn't what is actually building the binding that we are shipping. Samples can target anything as long as they support netcoreapp2.1